llm-complete-guide/run.py

```diff
@@ -239,7 +241,7 @@ def main(
             query_text,
             model=model,
             use_reranking=use_reranker,
-            model_version_stage="production",
+            # model_version_stage="latest",
```

Comment:
needed?

Reply:
nope, should be removed. this was one instance where i accidentally left this parameter in after the refactor. we dont take this in as an input anymore.